### ISSUE TITLE
kernel module compilation error.

### ISSUE THREAD

**Issue by sergey-safarov** [sipwise/rtpengine#974]
I want install module on latest kernel and get this compilation error.
```
DKMS make.log for ngcp-rtpengine-8.2.2.0-1.el8 for kernel 5.7.0-0.rc1.20200416git9786cab67457.1.el8.x86_64 (x86_64)
Fri Apr 24 10:39:48 UTC 2020
make: Entering directory '/usr/src/kernels/5.7.0-0.rc1.20200416git9786cab67457.1.el8.x86_64'
  AR      /var/lib/dkms/ngcp-rtpengine/8.2.2.0-1.el8/build/built-in.a
  CC [M]  /var/lib/dkms/ngcp-rtpengine/8.2.2.0-1.el8/build/xt_RTPENGINE.o
/var/lib/dkms/ngcp-rtpengine/8.2.2.0-1.el8/build/xt_RTPENGINE.c: In function ‘proc_create_user’:
/var/lib/dkms/ngcp-rtpengine/8.2.2.0-1.el8/build/xt_RTPENGINE.c:697:45: error: passing argument 4 of ‘proc_create_data’ from incompatible pointer type [-Werror=incompatible-pointer-types]
  ret = proc_create_data(name, mode, parent, ops, ptr);
                                             ^~~
In file included from /var/lib/dkms/ngcp-rtpengine/8.2.2.0-1.el8/build/xt_RTPENGINE.c:18:
./include/linux/proc_fs.h:74:31: note: expected ‘const struct proc_ops *’ but argument is of type ‘const struct file_operations *’
 extern struct proc_dir_entry *proc_create_data(const char *, umode_t,
                               ^~~~~~~~~~~~~~~~
/var/lib/dkms/ngcp-rtpengine/8.2.2.0-1.el8/build/xt_RTPENGINE.c: At top level:
cc1: warning: unrecognized command line option ‘-Wno-address-of-packed-member’
cc1: some warnings being treated as errors
make[1]: *** [scripts/Makefile.build:267: /var/lib/dkms/ngcp-rtpengine/8.2.2.0-1.el8/build/xt_RTPENGINE.o] Error 1
make: *** [Makefile:1732: /var/lib/dkms/ngcp-rtpengine/8.2.2.0-1.el8/build] Error 2
make: Leaving directory '/usr/src/kernels/5.7.0-0.rc1.20200416git9786cab67457.1.el8.x86_64'
```

**Comment by sergey-safarov** [comment]
Relevant fix for other project
https://lore.kernel.org/netdev/20191225172546.GB13378@avx2/

**Comment by sergey-safarov** [comment]
Think this is fixed at 4d08a700f1474beba7ec3efd960a98856c5c4aef
